abaddon does not build with `-DENABLE_RNNOISE=OFF`

With rnnoise support disabled (targetting a Linux distro that lacks the package, as well all of the CPUs that this build will be running on are far too slow to handle it), abaddon does not build successfully, I get the following error:
```
[ 95%] Building CXX object CMakeFiles/abaddon.dir/src/windows/profilewindow.cpp.o
[ 95%] Building CXX object CMakeFiles/abaddon.dir/src/windows/threadswindow.cpp.o
[ 96%] Building CXX object CMakeFiles/abaddon.dir/src/windows/voice/voicewindow.cpp.o
/home/techflash/archpower-extra-pkgs/abaddon/src/abaddon/src/windows/voice/voicewindow.cpp: In constructor ‘VoiceWindow::VoiceWindow(Snowflake)’:
/home/techflash/archpower-extra-pkgs/abaddon/src/abaddon/src/windows/voice/voicewindow.cpp:118:42: error: ‘class AudioManager’ has no member named ‘GetSuppressNoise’
  118 |     m_noise_suppression.set_active(audio.GetSuppressNoise());
      |                                          ^~~~~~~~~~~~~~~~
/home/techflash/archpower-extra-pkgs/abaddon/src/abaddon/src/windows/voice/voicewindow.cpp: In lambda function:
/home/techflash/archpower-extra-pkgs/abaddon/src/abaddon/src/windows/voice/voicewindow.cpp:120:35: error: ‘class AudioManager’ has no member named ‘SetSuppressNoise’
  120 |         Abaddon::Get().GetAudio().SetSuppressNoise(m_noise_suppression.get_active());
      |                                   ^~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/abaddon.dir/build.make:1549: CMakeFiles/abaddon.dir/src/windows/voice/voicewindow.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:122: CMakeFiles/abaddon.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
make: Leaving directory '/home/techflash/archpower-extra-pkgs/abaddon/src/build'
```

I have a small patch that may fix this issue.  If it works, I'll submit a PR fixing it shortly.

--- a/src/windows/voice/voicewindow.cpp
+++ b/src/windows/voice/voicewindow.cpp
@@ -115,10 +115,12 @@ VoiceWindow::VoiceWindow(Snowflake channel_id)
         UpdateVADParamValue();
     });
 
+#ifdef WITH_RNNOISE
     m_noise_suppression.set_active(audio.GetSuppressNoise());
     m_noise_suppression.signal_toggled().connect([this]() {
         Abaddon::Get().GetAudio().SetSuppressNoise(m_noise_suppression.get_active());
     });
+#endif
 
     m_mix_mono.set_active(audio.GetMixMono());
     m_mix_mono.signal_toggled().connect([this]() {

